### PR TITLE
Spelling and language

### PR DESCRIPTION
**Pull request by errhrtrth**
Orthographe et tournure de phrases, il reste encore du travail.

### CODE REVIEW

File: translations/fr.yml [hunk 1]
```diff
@@ -34,64 +34,64 @@ fr:
...
         <strong>Dévellopement Décentralisé</strong> - Avec plusieurs groupes de développeurs qui implémentent autant de logiciels compatibles avec le protocole Bitcoin Cash, le futur est assuré : Le développement du protocole de Bitcoin Cash est résistant aux attaques politiques et sociales. Aucun groupe ou projet ne peut en prendre le contrôle. La <a href="https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-ml" target="_blank">liste de diffusion bitcoin-ml</a> est le point d'entrée pour les nouvelles propositions et le centre de coordination entre les différentes équipes de développement.
     faq:
       - question: "Qu'est ce que Bitcoin Cash ?"
         answer:
           - |
-            Bitcoin Cash est l'argent électronique peer-to-peer pour Internet. Il est entièrement décentralisé, sans banque centrale et ne nécessite aucun tiers de confiance pour fonctionner.
-      - question: "Bitcoin Cash est-il different de 'Bitcoin' ?"
+            Bitcoin Cash est une monnaie virtuelle Pair-à-Pair. Elle est complétement décentralisée, sans banque centrale et n'a pas besoin de tiers pour fonctionner.
+      - question: "Bitcoin Cash est-il diffèrent de 'Bitcoin' ?"
```
Comment (MohamedKiouaz) [inline translations/fr.yml]: différent

File: translations/fr.yml [hunk 2]
```diff
@@ -34,64 +34,64 @@ fr:
...
       - |
         <strong>Dévellopement Décentralisé</strong> - Avec plusieurs groupes de développeurs qui implémentent autant de logiciels compatibles avec le protocole Bitcoin Cash, le futur est assuré : Le développement du protocole de Bitcoin Cash est résistant aux attaques politiques et sociales. Aucun groupe ou projet ne peut en prendre le contrôle. La <a href="https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-ml" target="_blank">liste de diffusion bitcoin-ml</a> est le point d'entrée pour les nouvelles propositions et le centre de coordination entre les différentes équipes de développement.
     faq:
       - question: "Qu'est ce que Bitcoin Cash ?"
         answer:
           - |
-            Bitcoin Cash est l'argent électronique peer-to-peer pour Internet. Il est entièrement décentralisé, sans banque centrale et ne nécessite aucun tiers de confiance pour fonctionner.
-      - question: "Bitcoin Cash est-il different de 'Bitcoin' ?"
+            Bitcoin Cash est une monnaie virtuelle Pair-à-Pair. Elle est complétement décentralisée, sans banque centrale et n'a pas besoin de tiers pour fonctionner.
```
Comment (MohamedKiouaz) [inline translations/fr.yml]: les majuscules à Pair-à-Pair sont pas justiées je pense

Comment (MohamedKiouaz) [reply translations/fr.yml]: Laisser "tiers de confiance" me semble meilleur pour la compréhension.

**Comment by zquestz** [comment]
There are some merge conflicts in your PR, can you please update your changes on top of the latest master? I believe we had a couple FR commits, and now master has changed. =)

**Comment by zquestz** [comment]
Going to close this. Many updates have happened to the french translation since this was posted.

**Pull request closed without merging**